app/models/organization.rb

```diff
@@ -11,7 +11,6 @@ class Organization < ApplicationRecord
 
   acts_as_followable
 
-  has_many :api_secrets, through: :users
```

Comment:
this relation currently does not make sense. As there's no join table here.
When accessed in the console it returns the following error:

```ruby
> org.api_secrets
ActiveRecord::HasManyThroughOrderError: Cannot have a has_many :through association 'Organization#api_secrets' which goes through 'Organization#users' before the through association is defined.
```